modular-docs-manual/content/topics/creating_procedure_modules.adoc

```diff
@@ -2,7 +2,16 @@
 
 = Procedure modules
 
-Procedure modules explain how to do something. A procedure module contains numbered, step-by-step instructions to help the user accomplish a single task. Sometimes those tasks include substeps. Procedure modules must include a title, a brief introduction, and one or more steps in the form of imperative statements. Procedure modules can also contain prerequisites, verification steps, and additional resources or next steps.
+Procedure modules explain how to do something. A procedure module contains numbered, step-by-step instructions to help users accomplish a single task. Sometimes those tasks include substeps. Procedure modules must include a title, a brief introduction, and one or more steps in the form of imperative statements. Procedure modules can also contain the following sections:
+
+* Limitations
+* Prerequisites
+* Verification or Verification steps
```

Comment:
Should we provide the Procedure section after Prerequisites and before Verification or is this too obvious?

Reply:
Fixed